src/vtkh/filters/Lagrangian.cpp

```diff
@@ -94,7 +94,8 @@ void Lagrangian::DoExecute()
       using vectorField_d = vtkm::cont::ArrayHandle<vtkm::Vec<vtkm::Float64, 3>>;
       using vectorField_f = vtkm::cont::ArrayHandle<vtkm::Vec<vtkm::Float32, 3>>;
 		  auto field = dom.GetField(m_field_name).GetData();
-      if(!field.IsType<vectorField_d>() && !field.IsType<vectorField_f>())
```

Comment:
This is also an old change. This needs to be `IsType` not `IsSameType` and is the fist reason for the test failure.